src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Windows/WindowsUtilizationExtensions.cs

```diff
@@ -62,11 +62,9 @@ public static IResourceMonitorBuilder AddWindowsPerfCounterPublisher(this IResou
     /// An extension method that creates a few OpenTelemetry instruments for system counters.
     /// </summary>
     /// <param name="builder">The builder.</param>
-    /// <returns>Returns the builder.</returns>
+    /// <returns>The builder.</returns>
```

Comment:
Let's use:

```
<returns>The value of <paramref name="builder" />.</returns>
```

This is the pattern I've been adopting throughout the repo, which is succint and clear.